tmon: Fix unknown types

This header file assumes that other header files are implicitly included,
but that doesn't happen with the toolchain we use. Therefore, include them
explicitly.

Signed-off-by: AlexTMjugador <AlexTMjugador@users.noreply.github.com>
---
diff -pur a/tools/thermal/tmon/tmon.h linux/tools/thermal/tmon/tmon.h
--- a/tools/thermal/tmon/tmon.h	2019-07-03 13:13:45.000000000 +0200
+++ b/tools/thermal/tmon/tmon.h	2019-07-12 23:08:28.407436432 +0200
@@ -16,6 +16,9 @@
  *
  */

+#include <sys/types.h> 
+#include <pthread.h>
+
 #ifndef TMON_H
 #define TMON_H
 
